src/Components/Main.js

```diff
@@ -3,34 +3,50 @@ import { perform } from '../API'
 import './Main.css';
```

Comment:
So I don't know JS all that strongly, can't promise this will be the most comprehensive review.